core/ts/core/Arc.ts

```diff
@@ -126,7 +126,8 @@ export class Arc extends EventEmitter {
     if (store) {
       inputs[name] = store.pojo;
     } else {
-      this.log.warn(`computeInput: "${storeName}" (bound to "${name}") not found`);
+      // TODO(sjmiles): is this ok or not?
```

Comment:
a question is - what should be done, if a connection in the recipe is not resolved? 
should RecipeBuilder fail to construct the recipe? should the Chef not run it?
let's discuss.

Reply:
Traditionally it would be expected to create the Store.